apps/ensemble-engine/src/main.ts

```diff
@@ -1,9 +1,23 @@
 import { NestFactory } from '@nestjs/core';
 import { EngineAppModule } from './ensemble-engine.module';
-import 'dotenv/config'
+import 'dotenv/config';
 
 async function bootstrap() {
   console.log('Starting Ensemble Engine...');
+  console.log('process.env.MONGODB_URI:', process.env.MONGODB_URI);
```

Comment:
Need to move those prints to the services where they are used/loaded. This way it's messy

Reply:
So let's just remove them first.
It's not needed to make it work and it's already warned for RPC when missing and printed when it's not missing.
So another PR should be done to manage the MongoDB URI the same way than RPC URL.